packages/shopify-codemod/README.md

```diff
@@ -55,6 +55,44 @@ a(() => {
 });
 ```
 
+### `global-assignment-to-default-export`
+
+Use magic to automatically transform global variable references into import statements.
+
+```sh
+jscodeshift -t shopify-codemods/transforms/global-assignment-to-default-export <file>
+```
+
+#### Example
+
+```js
+App.whatever = 1;
+
+// BECOMES:
+
+'expose App.whatever';
+export default 1;
+```
+
+### `global-reference-to-import`
+
+Use magic to automatically transform global variable references into import statements.
+
+```sh
+jscodeshift -t shopify-codemods/transforms/global-reference-to-import <file>
+```
+
+#### Example
+
+```js
+console.log(App.Components.UIList);
+
+// BECOMES:
+
+import UIList from 'app/components/ui_list';
```

Comment:
Again, might want to mention that this is really meant for our codebase more than anyone else's, in case anyone stumbles across this and wants to use it.